src/processing/run.py

```diff
@@ -0,0 +1,29 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+from processing.ast_parsers.tree_sitter_parsers import *
+
+parser = ArgumentParser()
+parser.add_argument("--input", type=str, required=True, help="Path to the folder with Python train/validation/test")
```

Comment:
Should update to something like "language's train/validation/test data" to not be about Python.

Reply:
Thanks!